dependencies.md

```diff
@@ -0,0 +1,19 @@
+<!-- @formatter:off -->
+# Dependencies
+
+## Compile Dependencies
+
+| Dependency                           | License           |
+| ------------------------------------ | ----------------- |
+| github.com/exasol/error-reporting-go | [MIT][0]          |
+| github.com/gorilla/websocket         | [BSD-2-Clause][1] |
+
+## Test Dependencies
+
+| Dependency                                  | License |
+| ------------------------------------------- | ------- |
+| github.com/stretchr/testify                 |         |
```

Comment:
Both test dependencies have MIT License (no idea if needed for test dependencies)

https://github.com/stretchr/testify/blob/master/LICENSE
https://github.com/testcontainers/testcontainers-go/blob/master/LICENSE

Reply:
We will add project keeper to the project, this will generate this automatically.